### PR TITLE
Add method to rewrite query parameters to our expected formats

### PR DESCRIPTION
**Pull request by mbryzek**

### CODE REVIEW

File: src/main/scala/io/flow/apidoc-json-validation/FormData.scala [hunk 1]
```diff
@@ -8,6 +9,71 @@ import play.api.libs.json._
...
+        parseEncoded(value)
+      )
+    )
+  }
+
+  /**
+    * Converts the specified js value into a url form encoded string,
+    * recursively through all types.
+    * 
+    * @oaram keys Keeps track of the top level keys we are parsing to
```
Comment (chrhicks) [inline src/main/scala/io/flow/apidoc-json-validation/FormData.scala]: `@param`

Comment (mbryzek) [inline src/main/scala/io/flow/apidoc-json-validation/FormData.scala]: fixed

File: src/test/scala/io/flow/apidoc-json-validation/FormDataSpec.scala [hunk 2]
```diff
@@ -22,6 +22,43 @@ class FormDataSpec extends FunSpec with Matchers {
...
+      Json.obj("number" -> Seq("1", "2"))
+    )
+  }
+
+  it("arrays") {
+    FormData.parseEncoded("key=val1&key=val2") should be(
+      Map("key" -> Seq("val1", "val2"))
+    )
+    
+    FormData.parseEncoded("key=val1&key=val2&foo=bar") should be(
```
Comment (ericluria) [inline src/test/scala/io/flow/apidoc-json-validation/FormDataSpec.scala]: Does this still work if the params are out of order, e.g. `key=val1&foo=bar&key=val2`?

Comment (mbryzek) [inline src/test/scala/io/flow/apidoc-json-validation/FormDataSpec.scala]: yes - test case added

**Comment by mikeclimbrock** [comment]
nice! lgtm

**Pull request merged**